wgpu-core/src/device/queue.rs

```diff
@@ -1054,238 +1054,247 @@ impl Global {
             // This avoids vulkan deadlocking from the same surface texture being submitted multiple times.
             let mut submit_surface_textures_owned = FastHashMap::default();
 
-            {
-                let command_buffer_guard = hub.command_buffers.read();
+            (|| {
```

Comment:
Could we use the block-break form that we do elsewhere instead of a IILE

Reply:
I did this for ease of authoring and validating that this works, but I'd also prefer consistency with the rest of the codebase. 🙂👍🏻

Reply:
Resolved with a force-push to 8836d7f.